charts/cp-schema-registry/values.yaml

```diff
@@ -86,5 +120,20 @@ prometheus:
     imageTag: 6f82e2b0464f50da8104acd7363fb9b995001ddff77d248379f8788e78946143
     imagePullPolicy: IfNotPresent
     port: 5556
+  serviceMonitor: # defines monitoring for a service when using prometheus operator
```

Comment:
You're adding a new `serviceMonitor` yaml block (and I agree).
But you seems to be "breaking" the `resources` attribute that already exist, cf below.
Before that, the resources was inside the `jmx` block, and now it's inside the `serviceMonitor` block.
=> you need to move it back

NB: to improve readability and maybe avoid that issue, you probably should also remove the empty line before `resources`

Reply:
ahlala bien vu, comment t'as repéré ça 😅

Reply:
Le talent je pense 😅

Reply:
#modestie-ma-première-qualité :)

Reply:
@OuesFa comment as tu testé le chart avant la PR? C'est bien que Flo est catch ca mais il ne fait pas s'attendre a une review de ce niveau ca prend beaucoup trop de temps et les blems d'indentations devraint être facilement vues avec un helm test ou lint.